spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -133,7 +138,10 @@
         }
       }
 
-      expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
+      expect(subject).to configure_splunk('system/indexes.conf').with(expected_attributes)
+    end
+    it 'should notify the splunk service to restart' do
```

Comment:
How important is it to add this test now? I can't actually check for that here since the resource to be notified is in another recipe which does not already have its own spec.